frontend/src/app/components/containers/complaints/complaints.tsx

```diff
@@ -18,6 +18,7 @@ import { selectCurrentOfficer } from "../../../store/reducers/officer";
 import UserService from "../../../service/user-service";
 import Roles from "../../../types/app/roles";
 import Option from "../../../types/app/option";
+import { setActiveTab } from "../../../store/reducers/app";
```

Comment:
simple change here, you've re-imported the same reducer more than once. you'll want to move `setActiveTab `to line 12 where you're you're importing `selectDefaultZone`